FragmentService/FragmentServiceInterface.php

```diff
@@ -46,6 +46,15 @@ public function buildCreateForm(FormMapper $form, FragmentInterface $fragment);
     public function buildForm(FormMapper $form, FragmentInterface $fragment);
 
     /**
+     * Gets content to render fragment.
+     *
+     * @param FragmentInterface $fragment
+     *
+     * @return array
```

Comment:
What should this array look like? 1-dimensional, 2-dimensional?

Reply:
actually this is a 1-dimensional array like in AbstractFragmentService::getContent method

Reply:
Then say it in the comments ;)